src/cpu/pred/conditional.cc

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2025 Technical University of Munich
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
```

Comment:
The licence of header and source files are different. IMO the header is correct but I'm not an expert in that